hack/lib/vars.bash

```diff
@@ -11,25 +11,29 @@ fi
 # shellcheck disable=SC1091,SC1090
 source "$(dirname "${BASH_SOURCE[0]}")/../../test/vendor/knative.dev/test-infra/scripts/e2e-tests.sh"
 
+# Adjust these when upgrading the knative versions.
 readonly KNATIVE_SERVING_VERSION="${KNATIVE_SERVING_VERSION:-v0.13.3}"
-readonly KNATIVE_SERVING_OPERATOR_VERSION="${KNATIVE_SERVING_OPERATOR_VERSION:-v0.13.3}"
 readonly KNATIVE_EVENTING_VERSION="${KNATIVE_EVENTING_VERSION:-v0.13.0}"
-readonly KNATIVE_EVENTING_OPERATOR_VERSION="${KNATIVE_EVENTING_OPERATOR_VERSION:-v0.13.3}"
+
+# Adjust these when cutting a new CSV.
+# TODO: Read these from metadata.
+export CURRENT_CSV="serverless-operator.v1.7.2"
+export PREVIOUS_CSV="serverless-operator.v1.7.3"
```

Comment:
should the values be swapped here ?

Reply:
Yes, they most definitely should be! Thanks!

Reply:
Please see how we infer the values on master branch now. This commit brought the changes: https://github.com/openshift-knative/serverless-operator/commit/2a9e1adc339c5af323b914533358f26bcfafb001